content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -210,6 +210,27 @@ When you introspect a database with the `db pull` command with the `prisma` rela
 
 When you apply changes to your Prisma schema with Prisma Migrate or `db push` with the `prisma` relation mode enabled, Prisma will not use foreign keys in your database.
 
+### Indexes
+
+One side effect of foreign key constraints in databases is that the database usually also creates an implicit index (for example [MySQL]([https://dev.mysql.com/doc/refman/8.0/en/constraint-foreign-key.html#:~:text=MySQL%20requires%20that%20foreign%20key%20columns%20be%20indexed%3B%20if%20you%20create%20a%20table%20with%20a%20foreign%20key%20constraint%20but%20no%20index%20on%20a%20given%20column%2C%20an%20index%20is%20created.](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html#:~:text=MySQL%20requires%20indexes,as%20described%20previously.))) for the columns you create them for. This is so "foreign key checks can be fast and not require a table scan". 
+
+As relation mode `prisma` does not use foreign keys when creating the table, there also no index created and without a manually added index, there is the danger for queries to require full table scans which can be slow or expensive (on database providers that bill per accessed row). To help our users to avoid that, Prisma renders a warning in Prisma Schema when relation mode `prisma` is enabled and a field that is used in an `@relation` does not have an index defined:
```

Comment:
```suggestion
When `relationMode="prisma"`, Prisma Migrate does not use foreign keys when creating the table.
The database does not create the implicit index.
Without a manually added index, there is the danger for queries to require full table scans which can be slow or expensive (on database providers that bill per accessed row, e.g. PlanetScale).
To avoid that, Prisma renders a warning in for the Prisma schema in the CLI and in the language-server, when `relationMode="prisma"` is enabled and field(s) used in an `@relation` do not have a corresponding index defined:
```

Reply:
I've added the part about Prisma Migrate as part of my rewording of this section